std/typecons.d

```diff
@@ -5141,7 +5141,16 @@ therefore avoiding the overhead of $(D new). This facility is unsafe;
 it is the responsibility of the user to not escape a reference to the
 object outside the scope.
 
-Note: it's illegal to move a class reference even if you are sure there
+The class's destructor will be called when the result of `scoped()` is 
```

Comment:
possessive for an 's' ending word is just a trailing apostrophe: `class' destructor`